### PR TITLE
Add bootstrap override option

### PR DESCRIPTION
**Pull request by ichorid**
This pull request removes `use_testnet` option and replaces it with more general `bootstrap_override` option, that allows to, well, override bootstrap servers list. It is very useful to build your own isolated testnets without hacking the code.
The corresponding new option is added to `tunnel_helper` and Tribler Twisted plugins (it seems not to have any effect for the latter, probably because of Dispersy endpoint interference).
Anyways, support for this option is required to proceed with building Jenkins performance tests.

p.s.
as a small bonus, there is a small change to `.gitignore` to ignore log file generated by `nosetests`.

p.p.s
monstrous regexp for checking IP addresses is a shameless copypaste from a popular Stackexchange answer.

### CODE REVIEW

**Comment by qstokkink** [comment]
I'm guessing this may screw with @devos50 's experiments.

File: twisted/plugins/tunnel_helper_plugin.py [hunk 1]
```diff
@@ -23,10 +24,48 @@
 from Tribler.Core.simpledefs import NTFY_TUNNEL, NTFY_REMOVE
 from Tribler.dispersy.tool.clean_observers import clean_twisted_observers
 
-
 # Register yappi profiler
 from Tribler.dispersy.utils import twistd_yappi
 
+def is_valid_ipv4(ip):
```
Comment (qstokkink) [inline twisted/plugins/tunnel_helper_plugin.py]: You can steal a shorter version from Dispersy:

https://github.com/Tribler/dispersy/blob/7a2893bb4110da79ec7300652aa7baa4afb40a91/util.py#L247

Comment (ichorid) [reply twisted/plugins/tunnel_helper_plugin.py]: I don't want to add another dependency on Dispersy. And I am not even sure that this check should be there. We got no policy on this kind of checks.

Comment (ichorid) [reply twisted/plugins/tunnel_helper_plugin.py]: Ok, changed that to use "inet_aton".

**Comment by ichorid** [comment]
BTW, @devos50  approved this change.

**Comment by ichorid** [comment]
There was an error in check for windows64. I guess it's a bug in the check, some kind of race condition. I filed it under #3634

**Comment by qstokkink** [comment]
retest this please

**Comment by devos50** [comment]
@ichorid can this be merged?

**Comment by ichorid** [comment]
Oops, put "Thanks everyone" message into pull request description...
Could someone with admin rights fix the message to "Add bootstrap override option" ?

**Comment by qstokkink** [comment]
@ichorid it's fine: no one reads the additional message anyway.

**Comment by ichorid** [comment]
@qstokkink, it's in the commit message on Git!
Sure, it demonstrates the nice atmosphere inside our team, but it's not very useful for those who read the Git log...

**Pull request merged**